cfgov/unprocessed/apps/owning-a-home/js/explore-rates/index.js

```diff
@@ -23,43 +23,22 @@ require( '../../node_modules/rangeslider.js' );
 require( './tab' );
 require( '../placeholder-polyfill' );
 
+var TEMPLATE_DIR = '../../templates/explore-rates/';
```

Comment:
Did the webpack update necessitate this change?

Reply:
The webpack handlebar loader `handlebars-loader!` wouldn't work if the path included a variable, but Steven's moves that to the webpack-config, which makes better sense. It's not specifically webpack 4 I presume so could be spun out of here into its own PR if he wants.

Reply:
Yeah, this was more to show that each app could have it's own webpack config. You definitely want to handle things like this in Webpack but I didn't have to do it here.